assembly/__tests__/env-storage.spec.ts

```diff
@@ -0,0 +1,70 @@
+// This file is aim to test the env storage functions which are external functions
+
+import { env } from '../env';
+import { resetStorage } from '../vm-mock';
+import { Address, Storage } from '../std';
+import { stringToBytes } from '@massalabs/as-types';
+
+const testAddress = new Address(
+  'AU12E6N5BFAdC2wyiBV6VJjqkWhpz1kLVp2XpbRdSnL1mKjCWT6oR',
+);
+
+beforeEach(() => {
+  resetStorage();
+});
+
+describe('Testing env storage functions', () => {
+  test('Testing del (assembly_script_delete_data)', () => {
```

Comment:
Feels weird when reading this:
- test + Testing
- nobody care/need of assembly_script_delete_data because it's an internal function

I have two suggestions:
```suggestion
  test('delete of a key', () => {
```
or
```suggestion
  it('deletes a key', () => {
```